src/main/java/it/gov/pagopa/pu/workflow/wf/ingestionflow/paymentsreporting/activity/NotifyPaymentsReportingToIufClassificationActivity.java

```diff
@@ -1,14 +1,11 @@
 package it.gov.pagopa.pu.workflow.wf.ingestionflow.paymentsreporting.activity;
 
 import io.temporal.activity.ActivityInterface;
-import it.gov.pagopa.payhub.activities.dto.classifications.Transfer2ClassifyDTO;
-
-import java.util.List;
+import it.gov.pagopa.payhub.activities.dto.classifications.PaymentsReportingTransferDTO;
 
 @ActivityInterface
 public interface NotifyPaymentsReportingToIufClassificationActivity {
 
-  void signalIufClassificationWithStart(Long organizationId, String iuf,
-                                        String outComeCode, List<Transfer2ClassifyDTO> transfers2classify);
+  void signalIufClassificationWithStart(String iuf, PaymentsReportingTransferDTO paymentsReportingTransferDTO);
```

Comment:
it's supposed to have a List of transfer given a IUF
see https://pagopa.atlassian.net/wiki/spaces/SPAC/pages/1339031693/Classificazione+incassi#3.3.-Rendicontazione
```suggestion
  void signalIufClassificationWithStart(String iuf, List<PaymentsReportingTransferDTO> paymentsReportingTransferDTO);
```